package.json

```diff
@@ -19,7 +20,7 @@
     "debug": "^2.2.0",
     "lodash": "^3.9.3",
     "mpns": "^2.1.0",
-    "node-cache": "^2.2.0",
+    "node-cache": "^2.1.1",
```

Comment:
>  The 2.2.0 version was revoked and is no longer available.

Ah, I see, it's fine to downgrade in that case. Could you please make the downgrade in a standalone commit and describe the reasons for downgrade in the commit message?